server/extension/extension.go

```diff
@@ -362,6 +415,11 @@ func NewProxy(targetURL string, config ProxyConfig) (*httputil.ReverseProxy, err
 			req.URL.Scheme = url.Scheme
 			req.URL.Host = url.Host
 			req.Header.Set("Host", url.Host)
+			req.Header.Del("Authorization")
+			req.Header.Del("Cookie")
```

Comment:
I think the docs about the `Cookie` header are technically wrong. I think the proxy extension, if it uses the same auth code as the other servers, probably handles `Authorization` as well. Either way, we should probably document that we drop these two headers.

Reply:
The drop is already documented:
```
...
The request sanitization will remove sensitive information from the
request like the `Cookie` and `Authorization` headers.
...
```